03 - Showcases & Templates/Vaults/Template_Hub.md

```diff
@@ -0,0 +1,49 @@
+---
+aliases: Template_Hub
+- 
+tags:
+- seedling
+- Template_Hub
+publish: true
+---
+
+# Template_Hub
+Author: [Hugo Sanots (Zektor)](Zektor)
+
+%% Add a description below this comment. It doesn't need to be long: one or two sentences should be a good start. 
+
+Longer descriptions are also welcome! A few things you could cover are: 
+- Who is this useful for?
+- Where to start?
+- What each folder is for?
+- How to use tags?
+- Are any plugins needed? - Add links to them from 02.01
+	- Are they included in the kit/starter vault?
+- Any custom CSS? 
+	- Add links to 04 - Community themes or 05 - CSS Snippets or any other 10 - Resources
+- Any ancilliary tools?
+%%
+Obsidian Template Hub started on the 13th of June 2022. It was set up to gather information on as many note templates as possible.
+
+Because every person have a different workflow, note structure, thinking process etc. This place was set up so every one can add the templates they use for others to try, test, adopt or reconfigure to fit the induvial needs.
+
+## Download 
+
+%% Add the link where people can download this vault or kit %%
+
+Forum: [Obsidian Forum](https://forum.obsidian.md/t/template-hub/39499)
+
+Link: [GitHub](https://github.com/llZektorll/Template_Hub)
+
+%% If there are any steps one needs to complete to download the starter vault or kit list them here in an ordered list (e.g. uncomment the next comment)
+%%
+%% To download this vault, you first need to complete the following steps
+1. Step 1
+2. Step 2
+%%
+
+%% Hub footer: Please don't edit anything below this line %%
```

Comment:
Could you please remove the footer from this note? This section will also get newly written by one of our scripts.

Reply:
done